### DISCUSSION TITLE
Can't connect to TTLock

### DISCUSSION THREAD

**Discussion by bgeza** [jbergler/hass-ttlock#121] [Q&A] [answered]
Hi,
could you please help me? I'm getting this message in the installation process, after entering my credentials that I use in TTLock mobile app
![image](https://github.com/jbergler/hass-ttlock/assets/114845078/473725d5-a82a-4e0a-97fd-c99977dd80c1)
thanks,
Geza

**Comment by bgeza** [comment, marked as answer]
Solved, there was a typo

**Comment by malanbary1** [comment]
I am facing the same issue can you please help I tried so many times but it gives me the same issue and I am very sure on the user and password

**Reply by bgeza** [reply]
for me the problem was on the previous window, where the ttlock clientid and secret had to be entered

**Reply by malanbary1** [reply]
when I add the ttlock from integration it shows me this window it dose not show me the clientid and secret, i did delete it and reinstall it from HACs but it keep showing me login window

**Comment by jbergler** [comment]
You can refer to https://github.com/jbergler/hass-ttlock/issues/55#issuecomment-1610036292 for details/docs on how home assistant manages credentials.

**Reply by malanbary1** [reply]
great now it's working !!! thank you
